src/scripts/enable_unattended_upgrades.sh

```diff
@@ -97,6 +99,28 @@ write_content_to_file() {
     fi
 }
 
+# Make sure needrestart is installed and runs automatically to avoid user intervention
+
+# Check if needrestart is installed
+if command -v needrestart &> /dev/null; then
```

Comment:
I think we should be consistent with the way we use to check if a package is installed. Use the same way as here https://github.com/dappnode/DNP_CORE/pull/142/files#diff-71cc34fad3de76a3e56243d89ce5c92701478529b1bc157611ef766ba1f1a1d4R64 or update the old way to this new one.